### DISCUSSION TITLE
Running and debugging locally

### DISCUSSION THREAD

**Discussion by ravircit** [formkiq/formkiq-core#206] [General]
How can I debug and run locally? what are cmd required options

**Comment by reganwolfrom** [comment]
Hey @ravircit,

If you want to build and run locally, the easiest way is through SAM CLI: https://docs.formkiq.com/docs/getting-started/quick-start#install-with-sam-cli

If you only need to run locally with docker, and are hoping to debug your app's integration with FormKiQ, you an use the local installation: https://docs.formkiq.com/docs/getting-started/quick-start-local

I hope that helps. Thanks!
